File: 7.py
#Warunki: x > 9; x % 2 = 0; x % 3 != 0; x = 99;

x = 99
a = 0 # Licznik liczb
for i in range(99):
    if (x > 9) and (x % 2 == 0) and (x % 3 != 0):
        print(x)
        a+=1
    x-=1

print('\nLiczb bylo: ' + str(a))

File: 7.cpp
#include <iostream>

using namespace std;

int main()
{
    int a = 0;
    for (int i = 99; i > 9; i--) {
        if(i % 2 == 0 && i % 3 != 0)
        {
            cout << i << endl;
            a++;
        }
    }
    cout << "Liczb bylo: " << a;
}
